src/server.js

```diff
@@ -147,6 +147,12 @@ module.exports = function (options = {}) {
 	server.listen(listenParams, () => {
 		if (typeof listenParams === "string") {
 			log.info("Available on socket " + colors.green(listenParams));
+			fs.chmod(listenParams, Helper.config.chmod, (err) => {
```

Comment:
We can't assume that `Helper.config.chmod` exists because people who update may not add the value to their config.